packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -79,6 +79,14 @@ module.exports = {
         github: '0oayfl0lW6xetjKjD5d5',
         google: '0oay75bnynuF2YStd5d5',
       },
+      /* Trex values:
+      baseUri: 'https://okta-dev-parent.trexcloud.com',
+      registrationPolicyId: 'reg3kwstakmbOrIly0g7',
+      idps: {
+        github: '0oa3jobx2bBlylNft0g7',
+        google: '0oa3jaktbqkiwCthn0g7',
+      },
+      */
```

Comment:
I decided to leave these in here as a block comment since we often need them to do testing locally, on Netlify, etc and I've found myself searching for them in previous commits.